src/coord/rmr/conn.c

```diff
@@ -151,17 +151,11 @@ void MRConnManager_ReplyState(dict *stateDict, RedisModuleCtx *ctx) {
     RedisModule_ReplyWithCString(ctx, key);
 
     // Get the value (array of connection state strings)
-    char **stateList = dictGetVal(entry);
```

Comment:
can we avoid adding unrelated changes? we can add another PR to do these other cleanups